packages/breadcrumb/src/BreadcrumbItem.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { styled } from '@project44-manifest/react-styles';
+
+export const StyledBreadcrumbItem = styled('li', {
+  d: 'flex',
+  alignItems: 'center',
+  color: '$text-secondary',
+
+  '& a': {
```

Comment:
Now using Link in BreadcrumbItem, it would be nice to strengthen this implicit targeting of the Link, but importing StyledLink just to style it seemed weird. Thoughts? E.g. 
```
import StyledLink from ... 
... 
 [`& ${StyledLink}`]: {
    ...
  }
```

Reply:
so theres two ways we can do this:

1. wrap the `Link` in `styled` in this file and apply styles that way.
2. follow the styling react component guides in stitches, where we add the `toString()` function on our components: https://stitches.dev/docs/styling#target-a-react-component
2. use the `Link` className for specificity.

I would say we do either one or two. Two probably being the most stitches way of doing it.

Reply:
Awesome! Done